### PR TITLE
✨ [PRTL-1185] legend in lolliplot svg

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

**Comment by Jephuff** [comment]
The legend in the png export is broken :(
protein_viewer-braf-2017-06-02

**Comment by alex-wilmer** [comment]
@Jephuff removed the flex stuff and seems to work now for both png and svg

File: src/packages/@ncigdc/containers/Lolliplot.js [hunk 1]
```diff
@@ -467,10 +467,7 @@ const LolliplotComponent = compose(
                   margins: { top: 20, right: 20, bottom: 20, left: 20 },
                   embed: {
                     right: {
-                      width: 350,
-                      margins: {
-                        left: -350
```
Comment (Jephuff) [inline src/packages/@ncigdc/containers/Lolliplot.js]: May add margins back with `right: 20` the legend goes right to the edge.

**Comment by alex-wilmer** [comment]
good call. I also added more margin to the left hand side of the svg which looks nicer

**Pull request merged**